pkg/app/pipectl/cmd/event/register.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package event
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pipe-cd/pipe/pkg/app/api/service/apiservice"
+	"github.com/pipe-cd/pipe/pkg/cli"
+)
+
+type register struct {
+	root *command
+
+	name string
+	data string
+}
+
+func newRegisterCommand(root *command) *cobra.Command {
+	r := &register{
+		root: root,
+	}
+	cmd := &cobra.Command{
+		Use:   "register",
```

Comment:
Not a strong opinion but "register" reminds me of the first registration. But actually this just adds an event, so I feel like "add" is more appropriate than "register".

Not sure what is conventionally standard, it's up to you.

Reply:
I have no strong opinions on this name, but still think that "register" is fine.

Reply:
"register event" is okay to me 😁

Reply:
Fine, no objection to that 👍